package.json

```
@@ -55,23 +55,23 @@
   },
   "peerDependencies": {
     "jest": ">= 27",
-    "svelte": ">= 3"
+    "svelte": ">= 3 < 4"
   },
   "devDependencies": {
     "@rollup/plugin-node-resolve": "^13.0.4",
-    "@sveltejs/adapter-static": "^1.0.0-next.18",
-    "@types/jest": "^27.0.0",
+    "@sveltejs/adapter-static": "^2.0.2",
+    "@types/jest": "^29.5.3",
     "cross-env": "^7.0.3",
-    "doctoc": "^2.0.0",
-    "jest": "^27.0.0",
-    "jest-circus": "^27.0.0",
-    "node-sass": "^6.0.0",
+    "doctoc": "^2.2.1",
+    "jest": "^29.6.1",
+    "jest-circus": "^29.6.1",
+    "node-sass": "^9.0.0",
```

I think it'd be better to use `sass` than `node-sass`. `node-sass` is deprecated and has declining usage. It's caused me some troubles while working on this project
```suggestion
    "sass": "^1.64.0",
```